scripts/tutorial.ipynb

```diff
@@ -667,35 +667,207 @@
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ht.zeros((3,4,), device='gpu')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Distributed Computing"
+    "Arrays on the same device can be seemlessly used in any heat operation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "tensor([[5., 5., 5., 5.],\n",
+       "        [5., 5., 5., 5.],\n",
+       "        [5., 5., 5., 5.]], device='cuda:0')"
+      ]
+     },
+     "execution_count": 39,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "a = ht.zeros((3,4,), device='gpu')\n",
+    "b = ht.ones((3,4,), device='gpu')\n",
+    "a + b"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Parallel Interactive Interpreter"
+    "However, performing operations on arrays with mismatching device will purposefully result result in an error (due to potentially large copy overhead)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "RuntimeError",
+     "evalue": "expected type torch.FloatTensor but got torch.cuda.FloatTensor",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mRuntimeError\u001b[0m                                  Traceback (most recent call last)\n",
+      "\u001b[0;31mRuntimeError\u001b[0m: expected type torch.FloatTensor but got torch.cuda.FloatTensor"
+     ]
+    }
+   ],
+   "source": [
+    "a = ht.full((3,4,), 4, device='cpu')\n",
+    "b = ht.ones((3,4,), device='gpu')\n",
+    "a + b"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Dos and Don'ts"
+    "It is possible, though, to explicitly move an array from one device to the other and back."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 41,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "tensor([[4., 4., 4., 4.],\n",
+       "        [4., 4., 4., 4.],\n",
+       "        [4., 4., 4., 4.]])"
+      ]
+     },
+     "execution_count": 41,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "a = ht.full((3,4,), 4, device='gpu')\n",
+    "a.cpu()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When writing code for GPUs only, you might quickly find it tedious to explicitly place every on the GPU by specifying the `device=` parameter. Hence, it is possible to set a default backend on which HeAT will work on."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ht.use_backend('gpu')"
```

Comment:
This should be set_default_device

Reply:
Should rename the functions to use_device() and get_device(). Will open an issue